webrender/src/clip_scroll_node.rs

```diff
@@ -454,6 +457,14 @@ impl ClipScrollNode {
         }
     }
 
+    fn clamp_to_bounds(
```

Comment:
It probably makes more sense for this to be a trait on f32 if possible. The self value is unused here.